vulture_os/system/cluster/views.py

```diff
@@ -142,7 +142,7 @@ def render_form(**kwargs):
         return render(request, 'system/cluster_edit.html', {'form': form, **kwargs})
 
     if request.method in ("POST", "PUT", "PATCH") and form.is_valid():
-        ip_changed = "management_ip" in form.changed_data
+        ip_changed = "management_ip" or "internet_ip" or "backends_outgoing_ip" or "logom_outgoing_ip"in form.changed_data
```

Comment:
This is not the condition you believe it is, it resolves to :
`True or True or True or "logom_outgoing_ip" in form.changed_data`